src/test/fuzz/coins_view.cpp

```diff
@@ -148,7 +149,8 @@ FUZZ_TARGET(coins_view, .init = initialize_coins_view)
                 }
                 bool expected_code_path = false;
                 try {
-                    coins_view_cache.BatchWrite(coins_map, fuzzed_data_provider.ConsumeBool() ? ConsumeUInt256(fuzzed_data_provider) : coins_view_cache.GetBestBlock());
+                    auto cursor = CoinsViewCacheCursor(cachedCoinsUsage, sentinel, coins_map);
+                    coins_view_cache.BatchWrite(cursor, fuzzed_data_provider.ConsumeBool() ? ConsumeUInt256(fuzzed_data_provider) : coins_view_cache.GetBestBlock());
```

Comment:
nit: now that you've extracted the first complex parameter, it might make the code slightly more readable if we extracted the second one as well